src/main/java/com/VetApp/PurrgentCare/services/AccountServiceImplementation.java

```diff
@@ -78,7 +79,15 @@ public AccountResponse associatePeople(AssociatePeopleWithAccountRequest associa
         var people = personRepository.findAllById(personIds);
         Account account = accountRepository.findById(accountId)
                 .orElseThrow(() -> new EntityNotFoundException(String.valueOf(accountId)));
-        account.setAccountHolders(people);
```

Comment:
Just setting the accountholders was causing an issue with the hibernate cascade.
Had to set the parent for each child and ignore duplicates or the add method would still add them.